notifications.bs

```diff
@@ -601,6 +604,47 @@ or vibrating the device again, unless the <a>renotify preference flag</a> is set
 </ol>
 
 
+<h3 id=selecting-an-icon-url>Selecting an icon URL</h3>
+
+<p>The user agent should select the most suitable icon while balancing multiple goals and
+constraints: high quality icon display, screen size, and availability of resources like memory and
+bandwidth.
+
+<p>The <dfn>icon URL selection steps</dfn> for selecting a URL given a list <var>icons</var> of type
+{{NotificationIconOptions}}, a <var>fallbackURLString</var> of type {{USVString}}, and
+<a lt="base url">base URL</a> <var>baseURL</var> are:
+
+<ol>
+  <li><p>Let <var>selectedURL</var> and <var>selectedIcon</var> be null.
+
+  <li><p>For each <var>icon</var> in <var>icons</var> run these substeps:
+
+    <ol>
+      <li>
+        <p>Let <var>iconURL</var> be the result of <a lt="url parser">parsing</a> <var>icon</var>'s
+        <code>src</code> using <var>baseURL</var>. If <var>iconURL</var> is failure, abort these
+        substeps for this <var>icon</var>.
```

Comment:
Now here the `<p>` does not need to be on a new line, since it's the only child.